packages/core/src/components/Modal/useShowHideModal.ts

```diff
@@ -37,22 +38,28 @@ export default function useShowHideModal({
     keys: KEYS
   });
 
-  // show/hide and animate the modal
   useEffect(() => {
-    if (!instance) {
```

Comment:
why did you removed this?

Reply:
The check is unnecessary here, it's already checked inside the conditions. If it exists here, when the modal is not being rendered, it will not render at all